### DISCUSSION TITLE
Loading animated sprite json using PixiJS v7.0.4

### DISCUSSION THREAD

**Discussion by shopper360mst** [pixijs/pixijs#8967] [Q&A]
im trying to get my animated sprite json loaded using Asset using PixiJS v7.0.4. 
I am able to get that working in 6.5 using asset loader.
now, Im getting an error unable to load the png stated within animated json file.  (the location of the json file loaded is stored in (../images/json/flag_anim.json) and it points to the location of the png which is (../images/flag/flag_anim.png). how do i go about this?

**Comment by SuperSodaSea** [comment]
Can you give some more detailed information, such as the content of your `flag_anim.json` and the error information?
